.github/workflows/build.yml

```diff
@@ -10,14 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         java-version:
-          - 8
-          - 9
-          - 10
-          - 11
-          - 12
-          - 13
-          - 14
-          - 15
+          - 16
```

Comment:
Not testing on Java 8 is annoying. Something to look into separately from this change.

Reply:
We can make the main tests run with java 8 via Gradle toolchains